test/realtime/tenants/replication_connection_test.exs

```diff
@@ -335,7 +339,7 @@ defmodule Realtime.Tenants.ReplicationConnectionTest do
           assert received_payload == payload
         end)
 
-      assert logs =~ "Unexpected relation on schema 'public' and table 'test'"
+      assert logs =~ "Recreating"
```

Comment:
Could we keep this test? We can start ReplicationConnection and then change the publication and still capture this case which is possible AFAIK.